pkgs/by-name/un/unimatrix/package.nix

```
@@ -0,0 +1,37 @@
+{ lib
+, python3Packages
+, fetchFromGitHub
+, git
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "unimatrix";
+  version = "unstable-2023-04-25";
+  format = "other";
+
+  src = fetchFromGitHub {
+    owner = "will8211";
+    repo = pname;
+    rev = "65793c237553bf657af2f2248d2a2dc84169f5c4";
+    hash = "sha256-MtHsoZvUpYPQ5O9CJC7TwcOQOT4ATfyNpoozWYSlI7Q=";
```

There was a [hash mismatch](https://github.com/NixOS/nixpkgs/pull/264716/checks?check_run_id=18545967268). I think it was caused because the `hash` attribute is the one from when `leaveDotGit = true;` was being used, which should cause a different hash.

The following suggestion sets the hash to the one returned by `nix-prefetch-git https://github.com/will8211/unimatrix.git 65793c237553bf657af2f2248d2a2dc84169f5c4`:

```suggestion
    hash = "sha256-fiaVEc0rtZarUQlUwe1V817qWRx4LnUyRD/j2vWX5NM=";
```